src/web/mjs/connectors/CypherScans.mjs

```diff
@@ -0,0 +1,13 @@
+import WordPressMangastream from './templates/WordPressMangastream.mjs';
+
+export default class CypherScans extends WordPressMangastream {
+
+    constructor() {
+        super();
+        super.id = 'cypherscans';
+        super.label = 'Cypher Scans';
+        this.tags = [ 'manga', 'manhwa', 'manhua', 'english' ];
+        this.url = 'https://cypherscans.xyz';
+        this.path = '/manga/list-mode';
```

Comment:
when fetched, manga list redirect to '/manga/list-mode/'. You may want to avoid that by add a trailing '/'